corporate/lib/support.py

```diff
@@ -192,10 +205,36 @@ def get_data_for_support_view(billing_session: BillingSession) -> SupportData:
             event_type=RemoteZulipServerAuditLog.REMOTE_SERVER_CREATED,
             server__id=billing_session.remote_server.id,
         ).event_time
+        mobile_users = (
+            RemotePushDeviceToken.objects.filter(server=billing_session.remote_server)
+            .distinct("user_id", "user_uuid")
+            .count()
+        )
+        mobile_pushes = RemoteInstallationCount.objects.filter(
+            server=billing_session.remote_server,
+            property="mobile_pushes_forwarded::day",
+            end_time__gte=timezone_now() - timedelta(days=7),
+        ).aggregate(total_forwarded=Sum("value", default=0))
+        mobile_data = MobilePushData(
+            mobile_users=mobile_users, mobile_pushes_forwarded=mobile_pushes["total_forwarded"]
+        )
```

Comment:
These should be the same as what we do in `remote_activity.py` for these counts.

Noting here that [Django's documentation for aggregation](https://docs.djangoproject.com/en/5.0/topics/db/aggregation/#aggregating-on-empty-querysets-or-groups) notes that the `default` argument for `Sum` is wrapping the aggregate function in coalesce.